src/pages/latest-rules.js

```diff
@@ -130,7 +130,7 @@ const LatestRules = ({ data, location }) => {
   return (
     <div className="w-full">
       <Breadcrumb isLatest />
-      <SearchBar toSearch />
+      {/* <SearchBar toSearch /> */}
```

Comment:
Add a comment with a link to the issue
https://github.com/SSWConsulting/SSW.Rules/issues/1068